.github/workflows/nightly-check-db.yml

```diff
@@ -0,0 +1,58 @@
+name: nightly-check-db
+
+on: [push]
+
+env:
+  PYTHONPATH: ${{ github.workspace }}
+
+jobs:
+  download-snapshot:
+    runs-on: ubuntu-latest
+    container: quarkchaindocker/pyquarkchain:mainnet1.1.1
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: create workspace
+      run: mkdir -p tmp/workspace
+
+    - name: Download DB snapshot
+      run: |
+        curl -s https://s3-us-west-2.amazonaws.com/qkcmainnet/data/sample-data.tar.gz --output data.tar.gz
```

Comment:
can combine this step with `Move data`, by directly unzip the files into the target directory